### ISSUE TITLE
[BUG] - cant join the world

### ISSUE THREAD

**Issue by SorryImFrosty** [Kikiisyourfriend/ProjectIsothermalExpert#72]
### **Confirm your current modpack version:**

```
1.08
```
### **Are you on a server? Or in Singleplayer?:**

```
Singleplayer
```

### **Description:**

```
im trying to open the world and it crash
```

### **Steps to reproduce:**

```
try to join the singleplayer world
```
### **Crash log (*if any*):**
> *Must use a code-pasting website such as Pastebin, or a Gist.*

```
https://pastebin.com/0xigL2XU
```

**Comment by SorryImFrosty** [comment]
UPDATE: Found out the cyclic it's breaking my world, tryied disabling it and got it back working, added again, placed a few item of it and started crashing again

**Comment by Kikiisyourfriend** [comment]
One of your Refined Storage auto crafts has a bugged recipe, probably due to a mod removal, that THAT is what the error log that you pasted shows. 🙂

**Comment by SorryImFrosty** [comment]
oof, u cant tell me what crafter have it right?

**Comment by SorryImFrosty** [comment]
nvm, just found out what is the bugged one

**Comment by itsjusttriz** [comment]
@SorryImFrosty We cannot. If removing Cyclic earlier temporarily fixed the issue, I would assume it's a recipe of an item from Cyclic. I would suggest temporarily removing Cyclic again, and as quick as possible, try to find the problematic recipe card and clear the recipe from it. Then put Cyclic back in and see if the issue continues.

**Comment by itsjusttriz** [comment]
Ah good. If all is good, I'll close this issue.

**Comment by SorryImFrosty** [comment]
it's not good but ok..
added the recipe for the pickaxe autocrafting in the crafter and crashed again after a item user requested it

**Comment by itsjusttriz** [comment]
Paste the log

**Comment by SorryImFrosty** [comment]
1 sec

**Comment by SorryImFrosty** [comment]
there is a way to show u live? like discord?

**Comment by itsjusttriz** [comment]
No. But if there's a crash log, that'll give us something to work with.

**Comment by SorryImFrosty** [comment]
ill'try  to see what is the bugged pickaxe and if it crash i'll send the log

**Comment by itsjusttriz** [comment]
Ok

**Comment by SorryImFrosty** [comment]
https://pastebin.com/aQCinzsy

**Comment by SorryImFrosty** [comment]
tryied adding the iron pickaxe in the autocrafting then placing a item user that reqeust it throw a exporter with a crafting upgrade and it crashed with that log

**Comment by SorryImFrosty** [comment]
https://pastebin.com/HbM6hS8x
tryied to recreate the pattern for the pickaxe but it's not gonna work anyway

**Comment by SorryImFrosty** [comment]
Ok after testing a few hours, no more problem, just using crafter instead of refined to autocraft and now it work, idk what's wrong with the pickaxe in the refined but oke
